utils/logger.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/xanzy/go-gitlab"
+)
+
+func Logger(project *gitlab.Project, mergeRequest *gitlab.MergeRequest) *log.Entry {
+	fields := log.Fields{
+		"project": project.NameWithNamespace,
```

Comment:
@mariusheine Also redundant

Reply:
Only logging the webUrl to reduce redundancy